plugins/parquet/README.md

```diff
@@ -78,4 +78,66 @@ To select the fields that you wish to partition your data on pass in a string of
-```
\ No newline at end of file
+```
+# Apache Parquet/Arrow to ECL Type Mappings
+
+| Apache Parquet / Arrow Type | ECL Equivalent | Notes |
+|----------------------------|-----------------|-------|
+| Boolean                     | BOOLEAN         |       |
+| Int8, Int16, Int32          | INTEGER         | Size depends on the Arrow type; ECL INTEGER has fixed size, typically 32 bits |
+| Int64                       | INTEGER8        |       |
+| UInt8, UInt16, UInt32       | UNSIGNED        | Size depends on the Arrow type; ECL UNSIGNED typically has a 32-bit size |
+| UInt64                      | UNSIGNED8       |       |
+| Float16 (Half Float)        | REAL4           | Stored as REAL4 in ECL; Precision may be lost due to conversion from 16-bit to 32-bit float |
+| Float                       | REAL4           | ECL REAL4 is a 32-bit floating-point number |
+| Double                      | REAL8           | ECL REAL8 is a 64-bit floating-point number |
+| Decimal128, Decimal256      | DECIMAL         | Precision and scale in ECL must match those in Parquet; ECL DECIMAL allows variable precision |
+| Date32, Date64              | UNSIGNED        | Stored as days or milliseconds since epoch; ensure proper date handling in ECL |
+| Time32, Time64              | UNSIGNED        | Stored as seconds or milliseconds since epoch; ECL UNSIGNED type can handle these formats |
+| Timestamp                   | UNSIGNED        | Stored as microseconds since epoch; may need conversion for precise handling in ECL |
+| Duration                    | RECORD          | Custom ECL structure may be needed to represent complex durations (e.g., days, hours, minutes) |
+| String, LargeString         | STRING, VARSTRING, UTF8 | ECL STRING or VARSTRING for variable-length text; UTF8 encoding may be used for compatibility |
+| Binary, LargeBinary         | DATA            | ECL DATA for binary data; handles raw byte storage |
+| FixedSizeBinary             | DATA            | Fixed-length binary data; ensure length matches between Parquet and ECL |
+| List, LargeList             | SET OF          | ECL SET OF to handle collections; `LargeList` corresponds to large collections in ECL |
+| Struct                      | RECORD          | Nested structures in ECL; ensure field definitions match Parquet schema |
+
+Note: When using `ParquetIO.read()`, ensure that the ECL record structure matches the Parquet schema. For complex types like Duration, you may need to define custom ECL structures.
+
+For more detailed information about Apache Arrow data types, refer to the [Apache Arrow Documentation](https://arrow.apache.org/docs/cpp/api/datatype.html).
+
+# Example Usage
+Here's a simple example demonstrating how to read and write Parquet files using basic data types:
+eclCopyIMPORT ParquetIO;
+
+```
+// Define a simple record structure
+PersonRecord := RECORD
```

Comment:
There are already a few examples of using the plugin, so it may be better to have an example with less native types like duration and timestamp.